### PR TITLE
Change font family from monospace to Droid Sans Mono

### LINKED ISSUE #1094: Customizable monospace font?
#1089 is great, but jarring since it's not the monospace font I use when editing. #179 added 'Operator Mono' but it seems to have disappeared. Perhaps there could be an option for people to specify an explicit font preference?

### PR DESCRIPTION
**Pull request by ganes1410**
Closes #1094

### CODE REVIEW

File: source/content.css [hunk 1]
```diff
@@ -316,7 +316,8 @@ followed someone
 /* Limit width of commit title and description inputs to 50/80 chars */
 #commit-summary-input,
 #commit-description-textarea {
-	font-family: monospace !important;
+  font-family: 'Droid Sans Mono', 'Operation Mono', 'Liberation Mono', 'Source Code Pro', Menlo, Consolas, Courier, monospace !important;
```
Comment (ramlmn) [inline source/content.css]: Shouldn't this be `'Operator Mono'` and not `'Operation Mono'`?

**Comment by ramlmn** [comment]
**A story of mine:** I sometimes design stuff and use different monospace fonts in different graphic designs, so I need to have all monospace fonts ([link](https://en.wikipedia.org/wiki/List_of_monospaced_typefaces), [link](https://www.fontsquirrel.com/fonts/list/classification/monospaced)) installed on my computer. But the main font I use for code is **Consolas** on **Windows**, and **Inconsolata** on **Ubuntu** (used system-wide with GNOME Tweak Tool).

The point is there is no specific `font-family` (monospace or otherwise) that satisfies everyone, someone is going to be disappointed. So the best font that we can use is `monospace` (because [**"browsers know best"**](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b)), and let the browser pick a font.

I think you can find some tool that lets you change default font in your operating system (if you are too paranoid to love a monospace font), or just use Stylish 🙃!

**Comment by ganes1410** [comment]
@bfred-it  what changes do i need to make to pass the tests?

**Comment by Ash258** [comment]
@ganes1410 https://travis-ci.org/sindresorhus/refined-github/builds/354725524#L3510
You have errors in your code (code style)

**Comment by ramlmn** [comment]
> The default GitHub font stack is better than `monospace` for most users

Then why change it (this PR)?

----------------

> If you want to have `monospace`

I don't, because GitHub's default font stack is the perfect one

------------

> resort to Stylish or #1186

You got me there 🤞!

-------------

*My opinion:* Don't change the default font stack provided by GitHub.

**Comment by bfred-it** [comment]
> My opinion: Don't change the default font stack provided by GitHub.

You're absolutely right. We had it at one point and then removed it https://github.com/sindresorhus/refined-github/pull/575

Let's stick to that decision.

Closing.

Thanks for this PR anyway @ganes1410, hope to see more from you :)

**Pull request closed without merging**